homeassistant/components/sensor/bmw_connected_drive.py

```diff
@@ -74,6 +83,16 @@ def unit_of_measurement(self) -> str:
         """Get the unit of measurement."""
         return self._unit_of_measurement
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the binary sensor."""
+        vehicle_state = self._vehicle.state
+
+        return {
+            'last_update': vehicle_state.timestamp,
```

Comment:
Please remove.

Reply:
See my comment above an this last update time.